openmc/mgxs/mgxs.py

```diff
@@ -212,11 +229,70 @@ def num_groups(self):
         return self.energy_groups.num_groups
 
     @property
+    def scores(self):
+        return ['flux', self.rxn_type]
+
+    @property
+    def filters(self):
+        group_edges = self.energy_groups.group_edges
+        energy_filter = openmc.Filter('energy', group_edges)
```

Comment:
I think it's ok as is. If this is just getting generated from the tallies property, *that* is saved in an attribute and wouldn't be generated again.